tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php

```diff
@@ -37,7 +37,8 @@ public function testsTraitAsserts()
         $this->makeAssertTest('assertEmptyArray', [[]], ['foo', [1], new \stdClass, [[]]]);
 
         /* @see AdditionalAssertionsTrait::assertIsString */
-        $this->makeAssertTest('assertIsString', ['foo', 'bar'], [null, 1, new class {
+        $this->makeAssertTest('assertIsString', ['foo', 'bar'], [null, 1, new class
```

Comment:
`{` in anonimous classes shold be on same line with class definition (CS rule)

Reply:
Done